nncf/quantization/algorithms/weight_compression/gptq.py

```diff
@@ -264,7 +264,7 @@ def _quantize_weights(
                         scales.append(scale)
                     else:
                         if self._scale_estimation and block_compression_config.num_bits == 4:
-                            activations = [inp.squeeze()[:, (i1 + i) : (i1 + i + group_size)] for inp in inputs]
+                            activations = [inp[..., (i1 + i) : (i1 + i + group_size)] for inp in inputs]
```

Comment:
Slice by last dimension, which is supposed to be hidden one.
It's aligned with processing statistics in `activations_to_wc_statistics` when reduction axes are all dimensions except the last one